src/trunk.c

```diff
@@ -460,7 +460,8 @@ typedef struct ONDISK trunk_super_block {
  */
 typedef uint16 trunk_subbundle_state_t;
 typedef enum trunk_subbundle_state {
-   SB_STATE_UNCOMPACTED_INDEX = 0,
+   SB_STATE_UNINIT = 0,
```

Comment:
- Because this state resides on disk, this change will cause a backward compatibility.